decidim-core/lib/decidim/core/test/shared_examples/authorable_interface_examples.rb

```diff
@@ -38,9 +38,22 @@
         model.update(user_group: user_group, author: nil)
       end
 
-      it "includes the user's ID" do
+      it "includes the user group's name" do
         expect(response["author"]["name"]).to eq(user_group.name)
       end
     end
+
+    describe "with an organization" do
+      let(:organization) { model.participatory_space.organization }
+      let(:query) { "{ author { name } }" }
+
+      before do
+        model.update(author: organization, user_group: nil)
+      end
+
+      it "do not return a main author" do
```

Comment:
```suggestion
      it "does not return a main author" do
```

Reply:
Nice catch! I copied this from `coauthorable_interface_examples.rb` so I also corrected that there.